src/snowflake/cli/api/project/definition.py

```diff
@@ -55,14 +55,14 @@ def load_project_definition(paths: List[Path]) -> ProjectDefinition:
 
 
 def generate_local_override_yml(
+    conn: SnowflakeConnection,
     project: ProjectDefinition,
 ) -> ProjectDefinition:
     """
     Generates defaults for optional keys in the same YAML structure as the project
     schema. The returned YAML object can be saved directly to a file, if desired.
     A connection is made using global context to resolve current role and warehouse.
```

Comment:
I need to edit these kinds of comments, since the connection is no longer "made using global context to resolve current role and warehouse".